### PR TITLE
Extend soon offline menu to seven days

### PR DESCRIPTION
**Pull request by mediaminister**
This fixes https://github.com/add-ons/plugin.video.vrt.nu/issues/388

### CODE REVIEW

**Comment by dagwieers** [comment]
I rebased this PR for testing.

File: resources/lib/apihelper.py [hunk 1]
```diff
@@ -93,24 +93,28 @@ def tvshow_to_listitem(self, tvshow, program, cache_file):
...
+                      page=None, items_per_page=None, use_favorites=False, variety=None, sort_key=None):
         """Construct a list of episode or season TitleItems from VRT NU Search API data and filtered by favorites"""
         # Caching
         if not variety:
             cache_file = None
-        elif use_favorites:
-            cache_file = 'my-{variety}-{page}.json'.format(variety=variety, page=page)
         else:
-            cache_file = '{variety}-{page}.json'.format(variety=variety, page=page)
+            cache_file = '{}{}{}.json'.format('my-' if use_favorites else '', variety, '-{}'.format(page) if sort_key is None else '')
```
Comment (dagwieers) [inline resources/lib/apihelper.py:103]: I understand what you are doing here, but I think it is becoming a bit clumsy.
I wonder if we could make it clearer by reformatting and still using kwargs.
```suggestion
            cache_file = '{my}{variety}{page}.json'.format(
                my='my-' if use_favorites else '',
                variety=variety,
                page='-{}'.format(page) if sort_key is None else '',
            )
```

Comment (mediaminister) [reply resources/lib/apihelper.py:103]: Thanks, this is better.

**Comment by dagwieers** [comment]
For me this works fine, and is probably good enough to fix #388

**Comment by mediaminister** [comment]
Okay, you can merge this if you're happy with this implementation.

**Comment by mediaminister** [comment]
I'll merge this.

**Pull request merged**